python/ray/serve/controller.py

```diff
@@ -345,6 +332,21 @@ async def run_control_loop(self) -> None:
             except Exception:
                 logger.exception("Exception updating application state.")
 
+            # Update the http proxy nodes set before updating the HTTP proxy states,
+            # so they are more consistent.
+            self._update_http_proxy_nodes()
```

Comment:
why rename from active_nodes -> http_proxy_nodes? This is a little bit of an abstraction leak

Reply:
I want to make it explicit that this one is for deciding the nodes where proxies should run: this is based on active nodes returned from deployment state manager and other factors (draining nodes).

Reply:
got it, makes sense